staticconf/config.py

```diff
@@ -439,6 +439,29 @@ def has_changed(self):
         return last_mtime < self.last_max_mtime
 
 
+def create_LoggingMTimeComparator(err_logger):
+    """Return an MTimeComparator subclass that calls err_logger(filename)
+    if os.path.getmtime(filename) fails."""
+
+    def try_getmtime_else_log(filename):
+        try:
+            return os.path.getmtime(filename)
+        except OSError:
+            err_logger(filename)
```

Comment:
Maybe also pass the exception just in case that can be used, to check errno, etc